Makefile

```diff
@@ -276,6 +270,15 @@ jstest: build-addons build-js-native-api-tests build-node-api-tests ## Runs addo
 		$(CI_JS_SUITES) \
 		$(CI_NATIVE_SUITES)
 
+.PHONY: jscov
+jscov:
+	$(RM) -r out/$(BUILDTYPE)/.coverage
+	$(MAKE) coverage-build-js
+	-NODE_V8_COVERAGE=out/$(BUILDTYPE)/.coverage \
+	  CI_SKIP_TESTS=core_line_numbers.js,testFinalizer.js,test_function/test.js \
+    $(MAKE) jstest
+	$(MAKE) coverage-report-js
```

Comment:
Just trying to understand this better. Even if the previous command fails, does it make sense to run this command?